harness/determined/pytorch/_callback.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Dict
+
+import torch
+
+
+class PyTorchCallback:
+    """
+    Abstract base class used to define a callback that should execute during
+    the lifetime of a PyTorchTrial.
```

Comment:
I think we should add a bit of documentation here that users should access callbacks via `context` object.

Reply:
The intention of the API is for users to access these through the getters, which are documented.